pghoard/pghoard.py

```diff
@@ -364,6 +366,16 @@ def _get_delta_basebackup_files(self, site, storage, metadata, basebackup_name_t
             if backup_name != basebackup_name_to_delete:
                 # Keep data file in case if there is still a reference from other backups
                 keep_hexdigests |= backup_hexdigests
+            else:
+                # Add bundles to remove
+                for chunk in meta.get("chunks", []):
+                    basebackup_data_files.append(
+                        os.path.join(
+                            self._get_site_prefix(site),
+                            FileTypePrefixes[FileType.Basebackup_delta_chunk],
+                            chunk["chunk_filename"],
+                        )
+                    )
```

Comment:
Codecov complains about coverage here, and this looks like something that should particularly be tested.